src/Evolution/Particles/MonteCarlo/TemplatedLocalFunctions.hpp

```diff
@@ -24,10 +24,13 @@ struct Packet;
 
 namespace detail {
 
-void draw_single_packet(gsl::not_null<double*> time,
-                        gsl::not_null<std::array<double, 3>*> coord,
-                        gsl::not_null<std::array<double, 3>*> momentum,
-                        gsl::not_null<std::mt19937*> random_number_generator);
+const double proton_mass_MeV = 938.272046;
```

Comment:
- Add the proton_mass to `PointwiseFunctions/Hydro/Units.hpp` in the same manner as neutron_mass
- remove this
- include and use `proton_mass` in the tpp file above